docs/api/input.md

```diff
@@ -99,6 +101,27 @@ import CSSProps from '@site/static/usage/v7/input/theming/css-properties/index.m
 
 <CSSProps />
 
+## Migrating from Legacy Input Syntax
```

Comment:
(Putting this in a random spot for threading purposes.) Now that the behavior of the `color` prop has changed, I wonder if we should include an explanation of what exactly it does for `ion-input` above the Colors playground. Especially on `ios` mode (which is the default), it can be hard to tell what's different between the various colors.

Reply:
Good point. I added this in https://github.com/ionic-team/ionic-docs/pull/2682/commits/0423590ea0c97d68ccbed22e680cf00eb4843856